pylti/flask.py

```diff
@@ -181,7 +184,7 @@ def is_role(self, role):
         else:
             raise LTIException("Unknown role {}.".format(role))
 
-    def _check_role(self):
+    def check_role(self):
```

Comment:
I think this should keep it's name `_check_role` and we should add back the pylint disable since this isn't ever meant to be exposed to users.